### PR TITLE
add mongo serveur auto-increment for version on node

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: ApiBundle/Controller/NodeController.php [hunk 1]
```diff
@@ -117,18 +116,10 @@ public function duplicateAction(Request $request, $nodeId)
...
-        $em = $this->get('doctrine.odm.mongodb.document_manager');
-        $em->persist($newNode);
-
-        $this->get('open_orchestra_backoffice.manager.node')->updateBlockReferences($node, $newNode);
-
-        $em->flush();
+        $documentManager = $this->get('doctrine.odm.mongodb.document_manager');
+        $documentManager->getConnection()->initialize();
+        $dataBase = $documentManager->getDocumentDatabase('OpenOrchestra\ModelBundle\Document\Node');
+        $dataBase->execute("db.loadServerScripts();duplicateNode({ nodeId: 'root', siteId: '2', language: 'en' });");
```
Comment (nicolasThal) [inline ApiBundle/Controller/NodeController.php]: I think this way is a little too linked to mongodb...

Comment (itkg-nanne) [inline ApiBundle/Controller/NodeController.php]: clearly yes! The previous flush too. We need to create a service with an interface. But this is just an example of something's working. We could test JMeter with it. If all is ok, create the service. Are you ok? @nicolasThal

Comment (nicolasThal) [inline ApiBundle/Controller/NodeController.php]: yes, you could test it on your vagrant first. @itkg-alavieille does jmeter work on the vagrant (maybe reduce the number of users)

**Comment by itkg-nanne** [comment]
Is it ok for you? @nicolasThal 
tests are coming...

**Comment by itkg-nanne** [comment]
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/381/files
https://github.com/open-orchestra/open-orchestra-model-interface/pull/96

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**